sphinx/source/docs/releases/0.12.15.rst

```diff
@@ -21,14 +21,33 @@ Grid Bounds
 
 Grids now automatically defer to any existing axis bounds when their
 own bounds are set to `"auto"`. Previously grids used always used the full
-range bounds. Thie change makes it simpler to have axes and grids both
+range bounds. This change makes it simpler to have axes and grids both
 use a consistent set of bounds.
 
 Minor Ticks
 ~~~~~~~~~~~
 
 Minor ticks are no longer displayed outside explicitly set axis bounds.
-Previously minor ticks would dislay one extra "major tick" distance outside
+Previously minor ticks would display one extra "major tick" distance outside
 explicit range bounds.
 
+Sphinx Plot Extension
+~~~~~~~~~~~~~~~~~~~~~
+
+The `bokeh-plot` Sphinx extension previously assumed the location of scripts was relative
+to a `source` subdirectory of the Sphinx project. This is no longer the case. Now, the full
+relative path from the Sphinx project (i.e. from the `Makefile`) should be provided.
+
+So what was once:
+
+.. code-block:: none
+
+    .. bokeh-plot:: docs/user_guide/examples/plotting_title_basic.py
+
+Should now be:
+
+.. code-block:: none
+
+    .. bokeh-plot:: source/docs/user_guide/examples/plotting_title_basic.py
```

Comment:
The part after "So what was once" is only true for a particular Sphinx project layout, which is common, but not required. I would just leave that part off.

Reply:
Fair enough, I was just trying to provide an example of what an example change might have to be.

Reply:
If you want to keep it we'd put need to be more specific, e.g. "For example, if your sphinx project has a layout like...."

Reply:
If you don't think it is necessary then removing it is fine.